configs/material-ui.json

```diff
@@ -15,14 +22,17 @@
     "lvl2": ".markdown-body h2",
     "lvl3": ".markdown-body h3, .markdown-body table td:first-of-type",
     "lvl4": ".markdown-body h4",
-    "text": ".markdown-body p, .markdown-body li,.markdown-body .home-text-wrap, .markdown-body table td:last-child"
+    "text": ".markdown-body p, .markdown-body li, .markdown-body table td:last-child"
   },
-  "selectors_exclude": [
-    ".anchor-link-style"
-  ],
   "only_content_level": true,
   "conversation_id": [
     "357227758"
   ],
+  "custom_settings": {
+    "attributesForFaceting": [
```

Comment:
If you want to leverage these two attributes, you need to implement these meta tags within your pages:

```HTML
<meta name="docsearch:language" content="en">
<meta name="docsearch:version" content="1.0.0">
```
See [more context here](https://community.algolia.com/docsearch/recommended-configuration.html#introduces-global-information-as-meta-tags)

Reply:
I was confused by https://community.algolia.com/docsearch/config-file.html#using-regular-expressions:

> The beneficial side effect of using this syntax is that every records extracted from pages matching http://www.example.com/docs/en/latest will have attributes lang: en and version: latest.